src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2750,22 +2785,26 @@ msgstr ""
 "Übersetzung (in diesem Fall {source_language}) nicht existiert."
 
 #: views/pages/page_sbs_view.py:99
+#, fuzzy
+#| msgid "You don't have the permission to edit this event."
 msgid "You don't have the permission to edit this page."
-msgstr "Sie haben nicht die nötige Berechtigung, um diese Seite zu bearbeiten."
+msgstr ""
+"Sie haben nicht die nötige Berechtigung, um diese Veranstaltung zu "
+"bearbeiten."
```

Comment:
all right, i will remove the the fuzzy comments, but i dont know how they could occur, because i did no changes to the corresponding template file

Reply:
Hmm, since this change was introduced in your latest [force-push](https://github.com/Integreat/cms-django/compare/9132464212a631df6417ed38c976f065767331ff..668f2455875477768659fd9923a0a532f2cd443c), maybe this somehow happened during your rebase or so... but I also don't understand completely how this could occur :thinking: